MeshLib/Elements/LineRule2.h

```diff
@@ -39,6 +39,9 @@ class LineRule2 : public EdgeRule
     /// Constant: The FEM type of the element
     static const CellType cell_type = CellType::LINE2;
 
+    /// Constant: The number of edges
+    static const unsigned n_edges = 1;
```

Comment:
Then it looks like a case for a separate PR plus some unit tests added, doesn't it?